cmake/CMakeLists.txt

```diff
@@ -1336,26 +1336,20 @@ set(temp_git_commit "(unknown)")
 set(temp_git_branch "(unknown)")
 set(temp_git_describe "(unknown)")
 set(temp_git_info "false")
-if(GIT_FOUND)
-  execute_process(COMMAND ${GIT_EXECUTABLE} describe HEAD
-    RESULT_VARIABLE temp_in_git_checkout
+if(GIT_FOUND AND EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/../.git)
```

Comment:
The other option and maybe the safer one would be to do ` ${GIT_EXECUTABLE} -C ${CMAKE_CURRENT_SOURCE_DIR}/../.git` everywhere.

Reply:
In case you where wonder over using `IS_DIRECTORY` instead of `EXISTS`, this was on purpose, as git submodules only have `.git` file not a directory. And lammps is used as a submodule e.g. in: https://gitlab.com/exaalt/exaalt